nntrainer/src/tensor.cpp

```diff
@@ -785,12 +785,12 @@ BroadcastInfo Tensor::computeBroadcastInfo(const Tensor &m) {
   BroadcastInfo e;
 
   for (unsigned int i = 0; i < MAXDIM; ++i) {
-    if (m_dim.getTensorDim(i) == 1) {
+    if (dim.getTensorDim(i) == m_dim.getTensorDim(i)) {
```

Comment:
I think it would be better to leave comment if the order is matter.

Reply:
I think we don't need to calculate the BroadcastInfo if both length is same to save calculation.

Reply:
Both done 👍